recipe/meta.yaml

```diff
@@ -13,22 +13,23 @@ source:
     - ncurses-5.9-gcc-5.patch
 
 build:
-  skip: true  # [win]
-  number: 0
-  detect_binary_files_with_prefix: true
+  skip: True  # [win]
+  number: 3
+  run_exports:
+    # pretty good compat within major version
+    #  https://abi-laboratory.pro/tracker/timeline/ncurses/
+    - {{ pin_subpackage('ncurses') }}
 
 requirements:
   build:
-    - toolchain
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
     - pkg-config
 
 test:
-  requires:
-    - pkg-config
-  # files:
-    # - run_test.sh         # [osx]
-    # Python test will not pass until it is recompiled against ncurses 6.1
-    # - run_test.py         # [osx]
```

Comment:
Understand that these have been a bit troublesome (hence commented). However am disappointed to see not only they are being dropped, but no significant testing of `ncuruses` is replacing them. Can we add some tests beyond trivial file tests? This library has a huge impact on user experience in the terminal and given how important this is would expect to see some testing that takes this into account.